api/AudioBufferSourceNode.json

```diff
@@ -0,0 +1,508 @@
+{
+  "api": {
+    "AudioBufferSourceNode": {
```

Comment:
The usual changes:
- ordering of browsers
- an interface level compat and spec entry.
- description field for constructor to make it clear
- no prefixing for property/method if it is at the interface level.